packer/registry_builder.go

```diff
@@ -50,6 +55,31 @@ func (b *RegistryBuilder) Run(ctx context.Context, ui packersdk.Ui, hook packers
 		log.Printf("[TRACE] failed to update HCP Packer registry status for %q: %s", b.Name, err)
 	}
 
+	go func() {
```

Comment:
I believe similar logic will be needed in the RegistryPostProcessor which wraps all configured post processors and run after a successful build has completed.

Reply:
Note: fixed in c31a436e7bdf13e34812baccc9ed2afd92945873